### ISSUE TITLE
Download links just 404

### ISSUE THREAD

**Issue by fancellu** [FPGAwars/iceInputs#1]
As above

https://github.com/FPGAwars/iceInputs/archive/v0.1.0.zip

https://github.com/FPGAwars/iceInputs/archive/master.zip

**Comment by Obijuan** [comment]
Yes,there is no stable release yet. You have to Download the dev version
